eldritch/items/unique.py

```diff
@@ -56,3 +58,228 @@ def get_trigger(self, event, owner, state):
+
+    if loss <= 0:
+      return None
+
+    reduction = events.LossPrevention(self, event, "stamina", 1)
+    return events.Sequence(
+        [
+            reduction,
+            events.AddToken(self, "stamina", event.character, n_tokens=1)
+        ],
+        owner
+    )
+
+
+class GateBox(Item):
+  def __init__(self, idx):
+    super().__init__("Gate Box", idx, "unique", {}, {}, None, 4)
+
+  def get_interrupt(self, event, owner, state):
+    if not isinstance(event, events.GateChoice) or not event.character == owner:
+      return None
+    if event.gate_name is None:
+      return None
+    return events.Sequence([
+        events.CancelEvent(event),
+        events.GateChoice(owner, "Choose any open gate", None, event.none_choice, event.annotation)
+    ],
+        owner)
+
+
+class HealingStone(Item):
+  def __init__(self, idx):
+    super().__init__("Healing Stone", idx, "unique", {}, {}, None, 8)
+
+  # Copied get_usable{_trigger,_interrupt,} from Physician ability
+  def get_usable_trigger(self, event, owner, state):
+    if not isinstance(event, events.UpkeepActions):
+      return None
+    return self.get_usable(event, owner, state)
+
+  def get_usable_interrupt(self, event, owner, state):
+    if not isinstance(event, (events.UpkeepActions, events.SliderInput)):
+      return None
+    return self.get_usable(event, owner, state)
+
+  def get_usable(self, event, owner, state):
+    if self.exhausted:
+      return None
+    if event.character != owner:
+      return None
+    # End copy-paste
+
+    available = [attr for attr in ["stamina", "sanity"] if getattr(owner, attr) < getattr(owner, f"max_{attr}")(state)]
+    if not available:
+      return None
+    elif len(available) == 1:
+      gain = events.Gain(owner, {available[0]: 1})
+    else:
+      gain = events.BinaryChoice(
+          owner,
+          "Gain 1 Stamina or 1 Sanity?",
+          "1 Stamina",
+          "1 Sanity",
+          events.Gain(owner, {"stamina": 1}),
+          events.Gain(owner, {"sanity": 1}),
+      )
+    # While the text doesn't explicitly say to exhaust, if you don't,
+    # you can keep gaining forever!
+    return events.Sequence([events.ExhaustAsset(owner, self), gain], owner)
+
+  def get_trigger(self, event, owner, state):
+    # The original card did not specify "Discard Healing Stone if the Ancient One awakens."
+    if isinstance(event, events.Awaken):
+      return events.DiscardSpecific(owner, [self])
+    return super().get_trigger(event, owner, state)
+
+
+class ObsidianStatue(Item):
+  def __init__(self, idx):
+    super().__init__("Obsidian Statue", idx, "unique", {}, {}, None, 4)
+
+  def get_usable_interrupt(self, event, owner, state):
+    if not isinstance(event, events.GainOrLoss) or not event.character == owner:
+      return None
+
+    types = [loss_type for loss_type in ("sanity", "stamina") if loss_type in event.losses]
+    seq = [events.DiscardSpecific(owner, [self])]
+    if len(types) == 1:
```

Comment:
Don't forget about losses of clues, where this item isn't usable.

Reply:
I figured that `loss_type in ("sanity", "stamina")` doesn’t include clues.